paper-server/src/main/java/org/bukkit/craftbukkit/block/CraftVault.java

```diff
@@ -1,17 +1,32 @@
 package org.bukkit.craftbukkit.block;
 
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableSet;
+import net.minecraft.resources.ResourceKey;
 import net.minecraft.world.level.block.entity.vault.VaultBlockEntity;
+import net.minecraft.world.level.block.entity.vault.VaultConfig;
 import org.bukkit.Location;
 import org.bukkit.World;
 import org.bukkit.block.Vault;
+import org.bukkit.craftbukkit.CraftLootTable;
+import org.bukkit.craftbukkit.inventory.CraftItemStack;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.loot.LootTable;
+import org.jetbrains.annotations.Unmodifiable;
+import org.jspecify.annotations.NullMarked;
+import org.jspecify.annotations.Nullable;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
 
+@NullMarked
```

Comment:
This is necesary now in implementations?

Reply:
It's not necessary, but I added it so that I don't have to add NonNull everywhere in the implementation or get warnings about it everywhere.

Reply:
oh make sense, if is valid then i take note of that too for the warnings.